pkg/components/rook/component.go

```diff
@@ -39,11 +39,15 @@ type component struct {
 	RookNodeAffinity         string
 	Tolerations              []util.Toleration `hcl:"toleration,block"`
 	TolerationsRaw           string
-	AgentTolerationKey       string `hcl:"agent_toleration_key,optional"`
-	AgentTolerationEffect    string `hcl:"agent_toleration_effect,optional"`
-	DiscoverTolerationKey    string `hcl:"discover_toleration_key,optional"`
-	DiscoverTolerationEffect string `hcl:"discover_toleration_effect,optional"`
-	EnableMonitoring         bool   `hcl:"enable_monitoring,optional"`
+	AgentTolerationKey       string            `hcl:"agent_toleration_key,optional"`
+	AgentTolerationEffect    string            `hcl:"agent_toleration_effect,optional"`
+	DiscoverTolerationKey    string            `hcl:"discover_toleration_key,optional"`
+	DiscoverTolerationEffect string            `hcl:"discover_toleration_effect,optional"`
+	EnableMonitoring         bool              `hcl:"enable_monitoring,optional"`
+	CSIPluginNodeSelector    util.NodeSelector `hcl:"csi_plugin_node_selector,optional"`
+	CSIPluginNodeAffinity    string
```

Comment:
Why not `CSIPluginNodeSelectorRaw`?

Reply:
By tacit convention `*Raw` has been for those that can simply be converted from it's hcl (json) form to the yaml and provided as it is. But `*NodeAffinity` is a rook's format of specifying the selector like this: `key1=value1; key2=value2;`.

Reply:
I _think_ it would still be fine as `Raw` is sort of template-ready format, but I'm fine with it as it is :+1: